lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -798,7 +846,28 @@ void SpecialFunctionHandler::handleDefineFixedObjectFromExisting(
   assert(os);
   char *data = (char*)address;
   for (uint64_t i = 0; i < size; ++i) {
-    os->write8(i, data[i]);
+    os->write8(state, i, data[i]);
+  }
+}
+
+void SpecialFunctionHandler::handleInitConcreteZero(ExecutionState &state,
```

Comment:
What do you use this for?

Reply:
This is used in fd.c, line 236 it seems, for initializing a symbolic pmem file to just be concrete zeros, rather than copying from some file.

Reply:
Yup. Essentially, this way, when a new allocation is made, it is returned as all zeros, rather than as a symbolic array of any value. This is 1. technically more accurate (as it's the POSIX semantics for some cases) and 2. helps constrain execution better.